application/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/silinternational/wecarry-api
 
-go 1.12
+go 1.13
```

Comment:
Should `Dockerfile` and `Dockerfile-dev` be updated as well?

Reply:
good point, I guess so